### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by Meowvw**
*fixed submit button on matchEndPopup
*delete unnecessary lines

### CODE REVIEW

**Comment by tyrcho** [comment]
@Meowvw can you please fix the merge ? Here you commited code which does not compile ...

File: companion/src/main/scala/net/hearthstats/modules/ReplayHandler.scala [hunk 1]
```diff
@@ -46,10 +46,26 @@ class ReplayHandler(
       uiLog.info(s"Video replay of your match is saved in <a href='$u'>$p</a>")
     } else throw new IllegalArgumentException(s"Could not rename $f to $newFile")
 
+<<<<<<< HEAD
```
Comment (tyrcho) [inline companion/src/main/scala/net/hearthstats/modules/ReplayHandler.scala]: @Meowvw see ? here you need to fix these lines before we can merge the PR

**Comment by Meowvw** [comment]
sorry about that，I think there re several conflicts between my code and master's. is there a way of ignoring these conflicts？I think only the change of MatchEndPop is enough for fixing the submitButton problem

Thank you
Suchen Wang ( Suzy )

**Comment by tyrcho** [comment]
Well you can't just *ignore* conflicts :) You need to decide which part of the file is to be kept ...

I use git in command line, and when I get a conflict on a merge I solve it with 'git mergetool -y', usually with kdiff3 as the merge tool.

**Pull request merged**